core/src/main/scala/org/genivi/sota/core/VehicleUpdatesResource.scala

```diff
@@ -124,6 +128,7 @@ class VehicleUpdatesResource(db : Database, resolverClient: ExternalResolverClie
       (post & extractNamespace & pathEnd) { ns => queueVehicleUpdate(ns, vin) } ~
       (get & extractNamespace & pathEnd) { ns => pendingPackages(ns, vin) } ~
       (get & extractUuid & path("download")) { downloadPackage } ~
+      (get & extractNamespace & path("operationresults")) { ns => results(ns, vin) } ~
```

Comment:
Hmm, ok I didn't realize that. I'll make another PR for that. Cheers for the heads up!